Sources/Sentry/Profiling/SentryProfiledTracerConcurrency.mm

```diff
@@ -178,7 +178,9 @@
         _unsafe_cleanUpContinuousProfilerV2();
     } else if (internalTraceId != nil) {
 #    if !SDK_V9
-        if (sentry_isContinuousProfilingEnabled(hub.getClient)) {
+        SentryClient *_Nullable client = hub.getClient;
+        if (nil != client
+            && sentry_isContinuousProfilingEnabled(SENTRY_UNWRAP_NULLABLE(SentryClient, client))) {
```

Comment:
`l`: You could add an extra log message when the client is nil and keep the existing one.